index.html

```diff
@@ -209,6 +214,44 @@ <h3>
             implicit signals.
           </p>
         </aside>
+        <section>
+          <h4>Lifetimes</h4>
+          <p>
+            Every [=permission=] has a <dfn class="export" data-dfn-for="permission">lifetime</dfn>,
```

Comment:
This needs to get integrated into the algorithms in order to have an effect. You could say that after |lifetime| time since the permission state became "granted", the UA has new information about the user's intent, but that doesn't really seem intuitive. Or you could integrate the lifetime into the definition of "permission state".